### PR TITLE
PlanNodeStats add addInputTiming, getOutputTiming and finishTiming

### PR DESCRIPTION
**Pull request by jinchengchenghh**
New planNodeStats output
```
-- Project[4][expressions: (c0:INTEGER, ROW["c0"]), (p1:BIGINT, plus(ROW["c1"],1)), (p2:BIGINT, plus(ROW["c1"],ROW["u_c1"]))] -> c0:INTEGER, p1:BIGINT, p2:BIGINT
      Output: 2000 rows (154.34KB, 20 batches), Cpu time: 907.80us, Blocked wall time: 0ns, Peak memory: 2.00KB, Memory allocations: 40, Threads: 1, CPU breakdown: I/O/F (27.24us/872.82us/7.74us)
   -- HashJoin[3][INNER c0=u_c0] -> c0:INTEGER, c1:BIGINT, u_c1:BIGINT
      Output: 2000 rows (136.23KB, 20 batches), Cpu time: 508.74us, Blocked wall time: 0ns, Peak memory: 88.50KB, Memory allocations: 7, CPU breakdown: I/O/F (177.87us/329.20us/1.66us)
      HashBuild: Input: 100 rows (1.31KB, 1 batches), Output: 0 rows (0B, 0 batches), Cpu time: 41.77us, Blocked wall time: 0ns, Peak memory: 68.00KB, Memory allocations: 2, Threads: 1, CPU breakdown: I/O/F(40.18us/1.59us/0ns)
      HashProbe: Input: 2000 rows (118.12KB, 20 batches), Output: 2000 rows (136.23KB, 20 batches), Cpu time: 466.97us, Blocked wall time: 0ns, Peak memory: 20.50KB, Memory allocations: 5, Threads: 1, CPU breakdown: I/O/F (137.69us/327.61us/1.66us)
      -- TableScan[2][table: hive_table] -> c0:INTEGER, c1:BIGINT
         Input: 2000 rows (118.12KB, 20 batches), Raw Input: 20480 rows (72.79KB), Output: 2000 rows (118.12KB, 20 batches), Cpu time: 8.89ms, Blocked wall time: 10.00us, Peak memory: 80.38KB, Memory allocations: 262, Threads: 1, Splits: 20, DynamicFilter producer plan nodes: 3, CPU breakdown: I/O/F (0ns/8.88ms/4.93us)
      -- Project[1][expressions: (u_c0:INTEGER, ROW["c0"]), (u_c1:BIGINT, ROW["c1"])] -> u_c0:INTEGER, u_c1:BIGINT
         Output: 100 rows (1.31KB, 1 batches), Cpu time: 43.22us, Blocked wall time: 0ns, Peak memory: 0B, Memory allocations: 0, Threads: 1, CPU breakdown: I/O/F (691ns/5.54us/36.98us)
         -- Values[0][100 rows in 1 vectors] -> c0:INTEGER, c1:BIGINT
            Input: 0 rows (0B, 0 batches), Output: 100 rows (1.31KB, 1 batches), Cpu time: 3.05us, Blocked wall time: 0ns, Peak memory: 0B, Memory allocations: 0, Threads: 1, CPU breakdown: I/O/F (0ns/2.48us/568ns)
```

### CODE REVIEW

File: velox/exec/PlanNodeStats.h [hunk 1]
```diff
@@ -76,6 +76,15 @@ struct PlanNodeStats {
   /// Sum of output bytes for all corresponding operators.
   uint64_t outputBytes{0};
 
+  // Sum of addInput for all corresponding operators.
```
Comment (mbasmanova) [inline velox/exec/PlanNodeStats.h:79]: Sum of CPU, scheduled and wall times for addInput call for all ...

File: velox/exec/PlanNodeStats.cpp [hunk 2]
```diff
@@ -100,6 +103,9 @@ std::string PlanNodeStats::toString(bool includeInputStats) const {
     out << ", Physical written output: " << succinctBytes(physicalWrittenBytes);
   }
   out << ", Cpu time: " << succinctNanos(cpuWallTiming.cpuNanos)
+      << ", Add input cpu time: " << succinctNanos(addInputTiming.cpuNanos)
```
Comment (mbasmanova) [inline velox/exec/PlanNodeStats.cpp:106]: Maybe shorten a bit more readability and move to the end of the message

CPU time: ..., Blocked all time: ..., CPU breakdown: I / O / F (addInput / getOutput / finish)

**Review by mbasmanova — COMMENTED** [review]
@jinchengchenghh Thank you for the change.

Would you update the summary to show an example of the new output?

Would be nice to update the docs as well: https://facebookincubator.github.io/velox/develop/debugging/print-plan-with-stats.html 

CC: @rui-mo

**Comment by jinchengchenghh** [comment]
Addressed all the comments, can you help review again?  Thanks! @mbasmanova

File: velox/exec/tests/AggregationTest.cpp [hunk 3]
```diff
@@ -862,6 +862,8 @@ TEST_F(AggregationTest, partialAggregationMemoryLimit) {
           .customStats.at("flushRowCount")
           .sum,
       0);
+  std::cout << toPlanStats(task->taskStats()).at(aggNodeId).toString()
```
Comment (mbasmanova) [inline velox/exec/tests/AggregationTest.cpp:865]: Accidental change? Let's remove.

**Comment by jinchengchenghh** [comment]
Addressed all the comments. Could you help review again? Thanks! @mbasmanova

File: velox/docs/develop/debugging/print-plan-with-stats.rst [hunk 4]
```diff
@@ -208,14 +249,14 @@ TableScan also reports the number of splits assigned.
 
 .. code-block::
 
-	Raw Input: 20480 rows (72.31KB), Splits: 20
+	Raw Input: 20480 rows (72.79KB), Splits: 20
 
-Velox also measures CPU time and peak memory usage for each operator. This
-information is shown for all plan nodes.
+Velox also measures CPU time and the breakdown of CPU time which including addInput, getOutput and finish time,
```
Comment (mbasmanova) [inline velox/docs/develop/debugging/print-plan-with-stats.rst:254]: There are some typos. Overall, it might be better to split this paragraph.

```
Velox also measures CPU time, peak memory usage and total number of memory allocations for each operator. 

Cpu time: 8.89ms, Peak memory: 80.38KB, Memory allocations: 262

A breakdown of CPU time into addInput, getOutput and finish stages of the operator is also available. I/O/F below is a shortcut for addInput/getOutput/finish.

CPU breakdown: I/O/F (0ns/8.88ms/4.93us)
```

File: velox/docs/develop/debugging/print-plan-with-stats.rst [hunk 5]
```diff
@@ -225,6 +266,13 @@ printPlanWithStats shows this information as “Blocked wall time”.
 
 	Blocked wall time: 10.00us
 
+Some operators like TableScan may produce the dynamic filter, reports the plan node ids.
```
Comment (mbasmanova) [inline velox/docs/develop/debugging/print-plan-with-stats.rst:269]: TableScan doesn't produce dynamic filters. These are produced by HashProbe operator. This doc update seems unrelated to the changes in the PR. Would you extract it in a separate PR so we can get it right without blocking this PR?

Comment (jinchengchenghh) [reply velox/docs/develop/debugging/print-plan-with-stats.rst]: Sure, changed, only left CPU Time and CPU breakdown change in this document.

**Review by mbasmanova — APPROVED** [review]
Thanks.
